backend_kvm.go

```diff
@@ -146,14 +146,14 @@ func (b kvmBackend) KernelPath() (string, string, error) {
 }
 
 func (b kvmBackend) InitrdModules() []string {
-	return []string{"kernel/drivers/char/virtio_console.ko",
-			"kernel/drivers/virtio/virtio.ko",
-			"kernel/drivers/virtio/virtio_pci.ko",
-			"kernel/net/9p/9pnet.ko",
-			"kernel/drivers/virtio/virtio_ring.ko",
-			"kernel/fs/9p/9p.ko",
-			"kernel/net/9p/9pnet_virtio.ko",
-			"kernel/fs/fscache/fscache.ko"}
+	return []string{"virtio_console",
+			"virtio",
+			"virtio_pci",
+			"9pnet",
+			"virtio_ring",
+			"9p",
+			"9pnet_virtio",
+			"fscache"}
 }
```

Comment:
I suggest adding an extra commit on top, which removes the manual dependency handling above.
From a quick look - virtio, 9pnet and fscache can go.

Please mention explicitly why they are safe to remove in the commit message - X is a dependency of Y, etc.

Reply:
done on latets commit

Reply:
Did you forget `virtio` or it must be explicitly copied?

Skimming through the outstanding list - virtio and virtio_ring are lowest level modules required by anything virtio. Not sure why virtio_pci is in the list - suspect it's no (longer) needed. Perhaps something to fix with a follow-up patch (PR even).

Reply:
virtio is not dependency of any module hardcoded above as I looked in both Arch and Debian(although it's builtin there).

So I ignored its removal suspecting it will not be copied via dependency tracing (In case copy required).

It may be that virtio(or virtio_ring) is not dependent of any modules since it is builtin in kernel and modinfo only list external modules(just assumption).  if this is the case we can remove virtio(or virtio_ring) since this will be either builtin or listed in depends. But as of now, I am not sure. Could be future patch

Reply:
It is logical for the module to not be listed as dependency, if it's already built-in.

For example - virtio and virtio_ring are built when [CONFIG_VIRTIO is set](https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/tree/drivers/virtio/Makefile?h=v5.14-rc1) while virtio_console [explicitly selects it](https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/tree/drivers/char/Kconfig?h=v5.14-rc1#n94) amongst others, yet `modinfo virtio_console` shows no dependencies.

Finishing that yak shaving can happen at later stage. This MR is good as-is.

Reply:
yep we can clean at later stages